test/integration/lift_drag_system.cc

```diff
@@ -59,14 +73,16 @@ class LiftDragTestFixture : public ::testing::Test
 
 /////////////////////////////////////////////////
 /// Measure / verify force torques against analytical answers.
-TEST_F(LiftDragTestFixture, VerifyVerticalForce)
+TEST_P(VerticalForceParamFixture, VerifyVerticalForce)
 {
   using namespace std::chrono_literals;
+  ignition::common::setenv("IGN_GAZEBO_RESOURCE_PATH",
+                           PROJECT_SOURCE_PATH "/test/worlds/models");
```

Comment:
How about `common::joinPaths` for windows? (Below `sdfFile` as well)

Reply:
Done https://github.com/ignitionrobotics/ign-gazebo/pull/955/commits/aa26825053a65c20700291b9b988ff12de37704e